kwargs.jl

```diff
@@ -4,9 +4,23 @@ import Pkg
 
 include(joinpath(@__DIR__, "autodetect-dependabot.jl"))
 
-function kwargs(; coverage::Bool,
-                  force_latest_compatible_version::Union{Bool, Symbol},
+function kwargs(; coverage,
+                  force_latest_compatible_version,
                   julia_args::AbstractVector{<:AbstractString}=String[])
+    if coverage isa AbstractString
+        coverage = parse(Bool, coverage)
+    end
+    @assert coverage isa Bool
+
+    if force_latest_compatible_version isa AbstractString
+        res = tryparse(Bool, force_latest_compatible_version)
+        if res === nothing
+            res = Symbol(force_latest_compatible_version)
+        end
+        force_latest_compatible_version = res
+    end
+    @assert force_latest_compatible_version isa Union{Bool, Symbol}
```

Comment:
since we aren't splicing literals into source code anymore, we have to parse it ourselves

Reply:
Can you remove the use of `@assert` and instead do an `if` block with a `throw`? In future releases of Julia, `@assert` may be disabled by default.

Reply:
I figured it was a legit assert, because it shouldn't really be possible to get something else given how the code is written/used. But sure, a throw is probably still better here, switched.